tools/update_options.py

```diff
@@ -13,7 +13,7 @@
 def Help():
   debug.dprint("Usage: update_options [OPTIONS] ... [FILES]\n" + \
                "\n" + \
-               "Updates flml, bml, swml and adml files. If FILES is not specified, all .flml, .bml, .swml and .adml files in\n" + \
+               "Updates flml, and adml files. If FILES is not specified, all .flml, and .adml files in\n" + \
```

Comment:
Changes to this file seem somewhat tangential to this pull request.

Why were these changes made?